### PR TITLE
Updated file structure for related items / outfit.  

### PR DESCRIPTION
**Pull request by cayla-c**
Just a closet full of skeletons really.  Skeletons that are missing a lot of bones.  More bones are forthcoming.  @jfleming9357 @daniel-he-dev @mattBman23 

I should put an emoji apparently.

### CODE REVIEW

**Review by jfleming9357 — COMMENTED** [review]
Does this data need to be exported? Also this is a massive file, so you might consider separating the data (dummy related products data, dummy products data, etc.)

**Review by jfleming9357 — CHANGES_REQUESTED** [review]
You can only have one export default. You can change to something like `export const *name*` but I don't believe it will work as it currently exists. I would group all of these into arrays, then export them like that.

File: client/components/RelatedAndOutfit/Outfit.jsx [hunk 1]
```diff
@@ -1 +1,21 @@
-import React from 'react';
\ No newline at end of file
+import React from 'react';
+import exampleProductDetails12012 from './exampleData.js';
+
+class RelatedItems extends React.Component {
+  constructor(props) {
```
Comment (jfleming9357) [inline client/components/RelatedAndOutfit/Outfit.jsx:6]: Is this supposed to be named 'Outfit'?

Comment (cayla-c) [reply client/components/RelatedAndOutfit/Outfit.jsx:6]: Haha yes - this widget is two carousel widgets, one is for items related to whatever is displayed at the top, and another is where the user can select products to add to their "outfit".

**Review by jfleming9357 — APPROVED** [review]

**Comment by daniel-he-dev** [comment]
looks fine

**Comment by daniel-he-dev** [comment]
sorry clicked the wrong button

**Pull request merged**